### PR TITLE
Support multiple tags for loopTags

### LINKED ISSUE #74: loopTags not working with multiple tags
For example:

```js
loopTags: ['for', 'each']
```

```html
<each loop="item in array">
  {{ item }}
</each>

<for loop="item in array">
  {{ item }}
</for>
```

Currently it doesn't work, because the plugin only looks at the first tag in `loopTags`, it sees that it's `for`, and it fails.

Working on a PR right now.

### PR DESCRIPTION
**Pull request by cossssmin**
This PR makes it possible to actually define multiple tag names for `loopTags`:

```js
loopTags: ['for', 'each']
```

```html
<each loop="item in array">
  {{ item }}
</each>

<for loop="item in array">
  {{ item }}
</for>
```

If merged, closes #74.

### CODE REVIEW

File: test/fixtures/loop_customtag.html [hunk 1]
```diff
@@ -1,5 +1,8 @@
 <p>x</p>
```
Comment (anikethsaha) [inline test/fixtures/loop_customtag.html:1]: a failing test would be great if not already there. 
else , Looks good  👍

Comment (cossssmin) [reply test/fixtures/loop_customtag.html:1]: failing test is in a7c6ef3d0aafc969284b9e96c5cd71bd54f22f70

Comment (anikethsaha) [reply test/fixtures/loop_customtag.html:1]: I meant test to show `the "${tagName}" tag must have a "loop" attribute` thing.

Comment (cossssmin) [reply test/fixtures/loop_customtag.html:1]: ah, that was a bug introduced by this PR (in 9874a8d) - didn't add a failing test, just fixed it directly...

Comment (anikethsaha) [reply test/fixtures/loop_customtag.html:1]: ohh ok cool ,

File: lib/index.js [hunk 2]
```diff
@@ -204,7 +204,7 @@ function walk (opts, nodes) {
     }
 
     // if the node has content, recurse (unless it's a loop, handled later)
-    if (node.content && node.tag !== loops[0] && node.tag !== scopes[0]) {
+    if (node.content && !loops.includes(node.tag) && node.tag !== scopes[0]) {
```
Comment (Scrum) [inline lib/index.js:207]: Explicit comparisons will always read better and work faster
```js
loops.includes(node.tag) === false
```

File: lib/index.js [hunk 3]
```diff
@@ -318,10 +318,11 @@ function walk (opts, nodes) {
     }
 
     // parse loops
-    if (node.tag === loops[0]) {
+    if (loops.includes(node.tag)) {
       // handle syntax error
       if (!(node.attrs && node.attrs.loop)) {
-        throw new Error(`the "${loops[0]}" tag must have a "loop" attribute`)
+        const tagName = loops.find(() => node.tag)
+        throw new Error(`the "${tagName}" tag must have a "loop" attribute`)
```
Comment (Scrum) [inline lib/index.js:325]: There is no need to search for a tag when it is known
```js
        throw new Error(`the "${node.tag}" tag must have a "loop" attribute`)
```

**Comment by Scrum** [comment]
@cossssmin @anikethsaha Hi guys, Sorry for taking so long

**Comment by Scrum** [comment]
@cossssmin Could you still add a description to the documentation as a separate PR?

**Comment by cossssmin** [comment]
> @cossssmin Could you still add a description to the documentation as a separate PR?

Sure thing, sorry about that!

**Comment by cossssmin** [comment]
Should I also add docs for the other options, like `scopeTags`?

**Comment by Scrum** [comment]
If it will not be difficult for you

**Comment by Scrum** [comment]
@cossssmin publish v1.4.0

**Pull request merged**